### ISSUE TITLE
The JWT was issued in the future

### ISSUE THREAD

**Issue by bartimaeus** [okta/okta-oidc-js#199]
I am using [okta-react](https://github.com/okta/okta-oidc-js/tree/master/packages/okta-react) for a single page app's sign in. On the backend I am using [okta-sdk-node](https://github.com/okta/okta-sdk-nodejs) to verify the token and provide other services.

I have users getting the following response after signing in to Okta with either their username/password or Google account:
![selection_010](https://user-images.githubusercontent.com/403832/39611320-16943866-4f14-11e8-9179-927bd107acf4.png)

```json
{ 
  "o": {
    "name": "OAUTH_ERROR",
    "message": "The JWT was issued in the future"
  }
}
```

I was thinking the issue is a timezone difference between the client and my node backend. But it looks like this is happening with the okta signin widget before any requests are made to my backend service.

Is there anyone else who has run into this issue? Thanks!

**Comment by shubhmali** [comment]
Yes setting system's time to automatically update based on your time zone helped.
Thank you.
